### PR TITLE
Create flexible document name in footer

### PR DESCRIPTION
**Pull request by chrisumphlett**
I edited the .tex file to utilize a parameter called footer_docname that is set in the YAML header in the .Rmd file. Then I updated the sample file to demonstrate how it would be used.

### CODE REVIEW

File: inst/rmarkdown/templates/awesomecv/resources/awesome-cv.tex [hunk 1]
```diff
@@ -136,9 +136,12 @@
 
 % Print the footer with 3 arguments(<left>, <center>, <right>)
 % Leave any of these blank if they are not needed
+% 2019-02-14 Chris Umphlett - add flexibility to the document name in footer, rather than have it be static Curriculum Vitae
 \makecvfooter
   {$date$}
-  {$name$~~~·~~~Curriculum Vitae}
+  $if(email)$
```
Comment (mitchelloharawild) [inline inst/rmarkdown/templates/awesomecv/resources/awesome-cv.tex]: `email` should be `footer_docname`
```suggestion
  $if(footer_docname)$
```

File: vignettes/sample.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@ github: mitchelloharawild
 linkedin: mitchelloharawild
 twitter: mitchoharawild
 headcolor: 414141
+footer_docname:: CV/Resume
```
Comment (mitchelloharawild) [inline vignettes/sample.txt]: I'm not sure if footer_docname is the best name for this. It is certainly descriptive for the awesomecv template, however elements in the YAML header should apply to all templates, which may include similar text in places other than the footer. Perhaps `docname` is better?

Also only one colon is required here.
```suggestion
footer_docname: CV/Resume
```

File: inst/rmarkdown/templates/awesomecv/resources/awesome-cv.tex (hunk 1 above)
Comment (chrisumphlett) [reply inst/rmarkdown/templates/awesomecv/resources/awesome-cv.tex]: lol sorry. I did it correctly on my local copy but then goofed when I updated.

File: vignettes/sample.txt (hunk 2 above)
Comment (chrisumphlett) [reply vignettes/sample.txt]: You mentioned that in the email. I haven't looked at the .tex for the other templates, but they don't look (to me) like that would be used for resume's. But I didn't completely think it through how the YAML is the same across templates. I think that template specific YAML samples would be helpful for others so they can understand elements that do/do not apply.

**Comment by chrisumphlett** [comment]
do we have it all squared away now so it can  be closed? I'm looking forward to re-installing the dev version and updating my resume with this code.

**Comment by mitchelloharawild** [comment]
Sure. I'll add another issue for extending this functionality to other templates. Thanks for the PR.

**Pull request merged**